lib/shared/drive-constraints.js

```diff
@@ -294,7 +294,7 @@ exports.COMPATIBILITY_STATUS_MESSAGES = {
    * The image and drive compatibility is not recommended; happens when the
    * actual drive size is smaller than the image's recommended drive size.
    */
-  SIZE_NOT_RECOMMENDED: 'Not Recommended',
+  SIZE_NOT_RECOMMENDED: 'Unrecommended Size',
```

Comment:
The wording here sounds a bit weird to me, and it also doesn't convey the reason it's not recommended. Can we change this to something with more detail?

Reply:
@jhermsmeier AFAIK this warning is only ever displayed for images that have a 'recommended size' set in their metadata manifest file (which we used to document, but now don't, because we decided that the "online catalog" was going to be storing all that info instead), and the _only_ images I'm aware of that use the manifest file are the images from http://www.ev3dev.org/downloads/ (they were an "early adopter" of Etcher). I've just checked a freshly downloaded ev3dev image, and the only fields they set in their manifest are `name`, `version`, `url` & `supportUrl` (and they also provide a logo and a blockmap).

So while I agree that 'Unrecommended Size' sounds weird, I don't think anybody will ever actually see it ;-)
(Just a little bit of historical context for you :slightly_smiling_face: )

Reply:
I'm aware of the history; but still – what's the point if this is not going to be used? Also, I think we should do it nicely, otherwise we'll forget about changing this later.

Reply:
I think it's something that @Shou suggested before, but given that this has/will never be used, what if this "recommended size" stuff was just completely removed (obviously that'd need to be done in a separate PR to this one, so apologies for being OT, but this seems the best place ATM to discuss this).

Reply:
I guess there's a larger discussion surrounding this so I've made this issue: https://github.com/resin-io/etcher/issues/2017. I'll leave the message alone in this PR by rolling back that change and we should get this through faster.